runtimes/core/src/sqldb/val.rs

```diff
@@ -116,7 +120,11 @@ impl ToSql for PValue {
                         val.to_sql(ty, out)
                     }
                     _ => {
-                        if let Kind::Enum(_) = ty.kind() {
+                        if is_pgvector(ty) {
```

Comment:
Can this be after the Enum case?

Reply:
ah, sorry, thought I fixed them all, fixed